lib/pom.xml

```diff
@@ -45,6 +45,10 @@
                         <sourceRoot>${project.build.directory}/generated-sources/dgm</sourceRoot>
                         <executable>java</executable>
                         <arguments>
+                            <!-- Translator currently downcasts to non-exported types in a few cases, mainly to check types of identifiers, and there is no obvious alternative. -->
```

Comment:
I think that some cases could be rewritten but the new code would be much more verbose. I am not sure though, and it would take some time to investigate further.

Reply:
This is just a build-time tool, it should not matter.

Reply:
True, but I looked into it in [3f0928b](https://github.com/cloudbees/groovy-cps/pull/120/commits/3f0928b84c126f3ceb1b5fdc728fc6de6a826d96) and migrating to only use the public API is relatively straightforward. In general the most common issue was using `JCIdent` to get the fully qualified class name of an identifier and/or its type, but we can switch to using `Element.toString()` and `Element.asType()`. (Awkward to use `toString`, but we were using `Symbol.toString()` previously, and there does not seem to be a better way, so...)